egui/src/context.rs

```diff
@@ -563,6 +563,29 @@ impl Context {
         std::sync::Arc::make_mut(&mut self.memory().options.style).visuals = visuals;
     }
 
+    /// The `tolerance` will be used in Cubic Bezier Curve, Quadratic Bezier Curve flattening.
+    /// It is the maximum distance between the original curve and the flattened curve.
+    /// The larger the tolerance, the less points will be generated.
+    /// The default value is 0.1.
+    /// The smaller the tolerance, the more points will be generated.
+    /// But practically, it should be kept in the range of [0.001, 2.0].
+    /// Example:
+    /// ```
+    /// let (mut response, painter) =
+    ///     ui.allocate_painter(ui.available_size_before_wrap(), Sense::click());
+    /// painter.ctx().set_tolerance(1.0);
+    /// ```
+    pub fn set_tolerance(&self, tolerance:f32){
+        (&mut self.memory().options.tessellation_options).bezier_flattern_tolerence = tolerance;
+    }
+
+    /// The `epsilon` will be used when comparing two floats.
+    /// For example, when checking the curve crossed the base line or not, the `epsilon` is used.
+    /// The default value is 1.0e-5 for f32.
+    pub fn set_epsilon(&self, epsilon:f32){
+        (&mut self.memory().options.tessellation_options).epsilon = epsilon;
+    }
```

Comment:
I've added quick-access to `tessellation_options` in https://github.com/emilk/egui/pull/1175 so these won't be needed!

Reply:
I have just used `git cherry-pick` to merge #1175 into my branch and I'm working on changing my code. Not sure this is the right approach or not. `syntax_highlighting.rs` is reporting errors in my branch. I guess it has other dependencies. I'm trying to figure out where the issue is.
```
the trait bound `for<'a> &CodeTheme: Deserialize<'a>` is not satisfied
the following implementations were found:
  <CodeTheme as Deserialize<'de>>
required because of the requirements on the impl of `SerializableAny` for `&CodeTheme`r
```
line 161 and 163.

Reply:
Too headache to merge this PR only. instead, I created a new PR #1178 which is based on today's latest master branch and solved this issue.